apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/gradingPanel.html.ts

```diff
@@ -92,6 +96,51 @@ export function GradingPanel({
               </li>
             `
           : ''}
+        ${showAiSubmissionGroup && context === 'main'
+          ? html`
+              <li class="list-group-item align-items-center">
```

Comment:
If I change the submission group, is there any way I see what the original group (from AI) was? I'm wondering if there is a way to indicate that the original value was overridden by a human.

<img width="560" height="416" alt="Image" src="https://github.com/user-attachments/assets/0cfbfe9c-78d7-4450-94e0-f3e7f4d2c30c" />

<img width="2490" height="858" alt="Image" src="https://github.com/user-attachments/assets/29e3755e-2af5-47d1-8f2d-67bf3415db97" />

In that same line of 'how do I restore the AI value?', is there any reason to have a 'group all ungrouped' option here?

<img width="526" height="386" alt="Image" src="https://github.com/user-attachments/assets/57c0915d-7ad7-439d-a3e2-02f94af6a33c" />

- I can't seem to filter by 'ungrouped' either